README.md

```diff
@@ -1,6 +1,102 @@
 # Agriboard
+
+Agriboard is an administrative dashboard for the agriculture software stack. It is used to manage users and view advanced analytics.
+
+## Dependencies
+
+### NodeJS
+
+To get started, the first thing you're going to need NodeJS (version `8.9.0` or higher). You can download this [here](https://nodejs.org/en/)
+
+### Yarn
+
+You will also require Yarn to manage the javascript dependencies in the application. You can view installation instructions for your OS [here](https://yarnpkg.com/lang/en/docs/install/)
+
+To install dependencies simply run `yarn`
+
+**Important:** Please use yarn to manage your dependencies. **You should never run any of the following commands:**
+
+- `npm install`
+- `npm install <package>`
+- `npm install --dev <package>`
+- `npm remove <package>`
+
+Instead use:
+
+- `yarn`
+- `yarn add <package>`
+- `yarn add --dev <package>`
+- `yarn remove <package>`
+
+## Scripts
+
+### Development
+
+Getting started developing is easy, simply run:
+
+`npm start`
+
+Hot reloading is enabled, so there is no need to restart the server or reload the page when you make changes. They will automatically propogate to your browser, while maintaining the previous state of your application.
+
+### Production
+
+If you wish to build for production, you can use the command
+
+`npm run build`
+
+If you want to serve these files, you can install `http-server` via:
+
+`yarn global add http-server`
+
+and serve files using:
+
+`http-server dist/ -p 8080`
+
+### Style Guide
+
+Agriboard adheres to the Airbnb style guide which can be found [here](https://github.com/airbnb/javascript). It is recommended you read over this briefly to understand the reasoning behind the linting rules.

```

Comment:
Oh? This already has it's linting rules set up?

Reply:
All of the projects I initialized have the same linting rules setup according to the airbnb spec